librz/core/cmd/cmd_print.c

```diff
@@ -7288,4 +7224,58 @@ RZ_IPI RzCmdStatus rz_cmd_base64_decode_handler(RzCore *core, int argc, const ch
 	rz_cons_println((const char *)buf);
 	free(buf);
 	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_print_bitstream_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	st32 len = (st32)rz_num_math(core->num, argv[1]), skip = 0;
+	if (!rz_str_isnumber(argv[1]) || len < 0) {
+		RZ_LOG_ERROR("len is not a positive number\n");
+		return RZ_CMD_STATUS_ERROR;
+	}
+	if (argv[2]) {
+		skip = (st32)rz_num_math(core->num, argv[2]);
+		if (!rz_str_isnumber(argv[2]) || skip < 0) {
+			RZ_LOG_ERROR("skip is not a positive number\n");
+			return RZ_CMD_STATUS_ERROR;
+		}
+	}
```

Comment:
avoids the rz_str_isnumber check. We assume this is done by RzCmd parser, so that we don't need to do it for each command handler (actually this is not true at the moment, but we should do such check..)

also, use `int` for len and just do:
int len = (int)rz_num_math(.....)
int skip = (int)rz_num_math(...)

if you make skip with `default_value` it will always have a value even when not specified.